_posts/fr/2023-05-17-policy.md

```diff
@@ -63,6 +63,31 @@ h2:not(:first-of-type) { margin-top: 3em; }
 
 {% include specials/policy/fr/06-consistence.md %}
 
+## Ressources du réseau
+
+*Originally published in [Newsletter #257](/fr/newsletters/2023/06/28/#en-attente-de-confirmation-7--ressources-du-réseau)*
+
+{% include specials/policy/fr/07-ressources-du-reseau.md %}
+
+## La politique comme interface
+
+*Originally published in [Newsletter #258](/fr/newsletters/2023/07/05/#en-attente-de-confirmation-8--la-politique-comme-interface)*
+
+{% include specials/policy/fr/08-interface.md %}
+
+## En attente de confirmation #9 : Propositions de politique
+
+*Originally published in [Newsletter #259](/fr/newsletters/2023/07/12/#en-attente-de-confirmation-9--propositions-de-politique)*
+
+{% include specials/policy/fr/09-propositions.md %}
+
+## En attente de confirmation #10 : Impliquez-vous
+
+*Originally published in [Newsletter #260](/fr/newsletters/2023/07/19/en-attente-de-confirmation-10--impliquez-vous)*
```

Comment:
```suggestion
*Originally published in [Newsletter #260](/fr/newsletters/2023/07/19/#en-attente-de-confirmation-10--impliquez-vous)*
```

Reply:
Thank you for correcting my mistakes

Reply:
@Copinmalin no problem. Do you want to try rebasing https://github.com/bitcoinops/bitcoinops.github.io/pull/1229 on master to pull the recent fr PRs which will fix the builds?

For reference I locally `git fetch --all` to pull the latest, then checkout your current translation branch `git checkout Newsletter-261-translate-in-french-` (for fr 261), then `git rebase origin/master` to rebase the 261 branch on the latest changes from master. There may be conflicts depending on the state of your 261 changes which you can resolve and then finish the 261 translation. Ill then use `make preview` to iterate on changes and test links locally and `make production` to run more checks like the ones run by netlify and travis here.